Penumbra/UI/ModsTab/ModPanelDescriptionTab.cs

```diff
@@ -34,14 +36,37 @@ public void DrawContent()
         ImGui.Dummy(ImGuiHelpers.ScaledVector2(2));
 
         ImGui.Dummy(ImGuiHelpers.ScaledVector2(2));
+        var sharedTagsEnabled = _sharedTagManager.SharedTags.Count() > 0;

```

Comment:
Don't use `Count()` unless really necessary. This is a list, so it has a `Count` property. `Count()` *should* resolve to just providing that property, but I don't trust LINQ enough. There is very rarely a reason to use LINQ-Count() when you're not actively filtering something.

Reply:
Huh, didn't notice that I used the `Count()` function here and the property elsewhere. I probably have VS's autocomplete to thank for that. Will fix.